website/static/js/search.js

```diff
@@ -283,7 +283,8 @@ var ViewModel = function(params) {
         self.currentPage(1);
         self.category(alias);
         if (alias.name === 'SHARE') {
-            document.location = '/share/?' + $.param({q: self.query()});
+            var win = window.open(window.contextVars.shareUrl + 'discover?' + $.param({q: self.query()}), '_blank');
```

Comment:
@chrisseto, this opens SHARE in a new tab; is there a vulnerability that needs to be handled?